pkg/database/migrations.go

```diff
@@ -2521,6 +2521,19 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 				)
 			},
 		},
+		{
+			ID: "00119-UserReportFromNumber",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE sms_configs ADD COLUMN IF NOT EXISTS twilio_user_report_from_number VARCHAR(255) NOT NULL DEFAULT ''`,
```

Comment:
```suggestion
					`ALTER TABLE sms_configs ADD COLUMN IF NOT EXISTS twilio_user_report_from_number TEXT NOT NULL DEFAULT ''`,
```

Reply:
the existing twilio_from_number is still varchar(255), so I'm being consistent